.github/workflows/main.yml

```
@@ -0,0 +1,48 @@
+name: RHVoice-CI
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  androidBuild:
+    name: Build android app
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```

```suggestion
          java-version: 8
```

similar to: https://github.com/RHVoice/RHVoice/blob/770546edd05dc237f098c9f3867e1702bd521806/.github/workflows/codeql-analysis.yml#L44